client/webpack.config.cjs

```diff
@@ -1,6 +1,5 @@
 const { resolve } = require('node:path');
 
-const ESLintPlugin = require('eslint-webpack-plugin');
```

Comment:
running eslint during compilation is unnecessary with:
1. IDE integration
2. pre-commit lint
3. CI lint